docs/internals/deprecation.txt

```diff
@@ -30,6 +30,9 @@ details on these changes.
 
 * ``Field.remote_field.to`` attribute will be removed.
 
+* The ``on_delete`` argument for ``ForeignKey``, ``ForeignObject``, and

```

Comment:
`ForeignObject` is private undocumented API and there's no reason to change that in this PR. So I don't really think it should be mentioned here.